.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Maybe we should use proper headings instead of bold text?

Reply:
I think for issues the big headings are a bit too bold.

Reply:
Can't we just use smaller headings then?

Reply:
Sure, changed it.